conf_how_to.py

```diff
@@ -82,10 +82,10 @@
 
 
 """
-You can specify the build with help of variable `DEFAULT_OPTIONS` 
+You can specify the build with help of variable `options` 
 which has following data (from `infrastructure/build_scripts/build_runner.py`):
 
-`DEFAULT_OPTIONS` is the dictionary that contains the following properties:
+`options` is the dictionary that contains the following properties:
```

Comment:
args amd log should be added to
as well as verbose parameter

Reply:
It can be added as external commit after implementing in production scripts